src/webui/src/components/SlideBar.tsx

```diff
@@ -57,32 +55,26 @@ class SlideBar extends React.Component<SliderProps, SliderState> {
             method: 'GET'
         })
             .then(res => {
-                if (res.status === 200 && this._isMounted) {
+                if (res.status === 200) {
                     this.setState({ version: res.data });
                 }
             });
     }
 
     handleMenuClick = (e: EventPer) => {
-        if (this._isMounted) { this.setState({ menuVisible: false }); }
+        this.setState({ menuVisible: false });
         switch (e.key) {
             // to see & download experiment parameters
             case '1':
-                if (this._isMounted === true) {
-                    this.setState(() => ({ isvisibleExperimentDrawer: true }));
-                }
+                this.setState(() => ({ isvisibleExperimentDrawer: true }));
```

Comment:
The warning indicates that some async handlers are not cancelled.
We should cancel the handlers in unmounting callback instead of hiding the warning.